packages/react-liveness/src/components/FaceLivenessDetector/service/utils/blazefaceFaceDetection.ts

```diff
@@ -23,13 +23,15 @@ export class BlazeFaceFaceDetection extends FaceDetection {
 
   constructor(binaryPath?: string, faceModelUrl?: string) {
     super();
-    this.faceModelUrl = faceModelUrl;
+    this.faceModelUrl =
+      faceModelUrl ??
+      'https://cdn.liveness.rekognition.amazonaws.com/face-detection/tensorflow-models/blazeface/0.0.7/model/model.json';
```

Comment:
Can we store 0.0.7 in a constant so we can use it elsewhere?

Reply:
yep will do